src/web_main.py

```diff
@@ -197,4 +197,15 @@ def pull_down_queue_member(queue_id, action):
     return "", 204
 
 
+@app.route("/api/queues/<int:queue_id>/rename-queue", methods=["PUT"])
+@login_required
+def rename_queue(queue_id):
+    new_name = request.data
```

Comment:
Let's perform back-end side validation for the name as well:
* trim the name
* check that it doesn't exceed max length (which is specified in the config file)

Reply:
It is still possible to save a name with whitespace at the beginning or the end of the string